src/vs/workbench/electron-sandbox/desktop.contribution.ts

```diff
@@ -241,8 +241,8 @@ import { LinuxCustomTitlebarExperiment } from './parts/titlebar/titlebarPart.js'
 			},
 			'window.experimentalControlOverlay': {
```

Comment:
We originally had added `window.experimentalControlOverlay` as a way to test WCO on Linux. We should prepare to remove this setting and be only WCO supportive going forward for Linux. I would not overload this setting to be used now on macOS. It either works on macOS with WCO or we should not use WCO on macOS imho.

Reply:
Yeah I was debating it myself, I am good with not being a setting.